### ISSUE TITLE
YOLACT as a package?

### ISSUE THREAD

**Issue by sayez** [dbolya/yolact#343]
Hi guys!

I just discovered YOLACT and it seems to be an interesting tool, but I was wondering if it was planned to have this repository as a python package that could be installed with conda/pip?

This way, sub-parts of an opensource code can be imported independently as a library.

As an example, the official implementation of pifpaf -for keypoints detection- can be installed using the command: 
> **conda install openpifpaf**

It would be great to integrate your opensource code and functions in other projects.

If it is not planned to publish a stand-alone package, is there a library out there (that I may not have found :P) that implements YOLACT officially?

Thanks in advance and for your amazing work!

**Comment by dbolya** [comment]
A couple people in the community have been wanting that and have kindly started a pull request to get this repo into shape so that it can be transitioned into a pip package. You can track our progress here: #323, though updates might be a bit slow because I don't have much time to work on this project right now.

**Comment by breznak** [comment]
Yes, I'd like that too. Currently not supported, WIP.  tracked here #299

**Comment by sayez** [comment]
I look forward to the time it will be available! 
Thanks for your responses!

**Comment by subhacom** [comment]
Looking forward to this as well.

**Comment by breznak** [comment]
I've updated #323 and #456, with that, the code is already quite usable programatically (API) for inference. Still no proper python package, but I'll do that afterwards.
